### ISSUE TITLE
Trivial scaling question

### ISSUE THREAD

**Issue by hholst80** [muupan/async-rl#11]
The loss function `v_loss` is accumulated like

```
v_loss += (v - R) ** 2 / 2
```

but then it is scaled with `v_loss *= self.v_loss_coef` where `v_loss_coef` is 0.5 by default.

Is there a reason why we're scaling it twice, termwise and also the final sum?

**Comment by muupan** [comment]
By `(v - R) ** 2 / 2` I mean just dividing squared errors by 2, which I think is common, though I'm not sure if the authors also did so.

By `v_loss_coef` I mean a scaling factor for tuning the relative learning rate of v. One of the authors told me they multiplied the gradients of v by 0.5.

**Comment by hholst80** [comment]
It seems a bit non-standard to scale both terms and sum like this. I'm trying without the 0.5 sum scaling and just dividing the terms as you do.

**Comment by hholst80** [comment]
Ofc for a particular instance there could be reason to balance the two loss functions differently so the constants are good to have. I was just curious about their default values. Thanks for your clarification!
